libmscore/line.cpp

```diff
@@ -185,17 +185,21 @@ bool LineSegment::edit(EditData& ed)
                         qDebug("LineSegment::edit: no start/end segment");
                         return true;
                         }
+
+                  int startTrack { computeStartTrack(track, scope(), score()) };
+                  int endTrack { computeEndTrack(track2, scope(), score()) };
+
                   if (ed.key == Qt::Key_Left) {
                         if (ed.curGrip == Grip::START)
-                              s1 = prevSeg1(s1, track);
+                              s1 = prevSeg1(s1, track, startTrack, endTrack);
                         else if (ed.curGrip == Grip::END || ed.curGrip == Grip::MIDDLE)
-                              s2 = prevSeg1(s2, track2);
+                              s2 = prevSeg1(s2, track2, startTrack, endTrack);
                         }
                   else if (ed.key == Qt::Key_Right) {
                         if (ed.curGrip == Grip::START)
-                              s1 = nextSeg1(s1, track);
+                              s1 = nextSeg1(s1, track, startTrack, endTrack);
                         else if (ed.curGrip == Grip::END || ed.curGrip == Grip::MIDDLE) {
-                              Segment* ns2 = nextSeg1(s2, track2);
+                              Segment* ns2{ nextSeg1(s2, track2, startTrack, endTrack) };
```

Comment:
A space is missing here before the `{`.

Reply:
Yeah, Visual Studio likes to eat those spaces before the opening curly brace in an initializer. Will update together with the code for your other comment when we figured the other one out.